media/js/src/CollectionTab.jsx

```diff
@@ -56,6 +57,18 @@ export default class CollectionTab extends React.Component {
         });
     }
 
+    onClearFilter() {
+        this.setState({
```

Comment:
This is working correctly for page 1. If I attempt to "Clear" on page > 1 then things get a bit confused.